content/posts/blog/first-paris-xmpp-meetup.md

```diff
@@ -0,0 +1,13 @@
+---
+title: First Paris XMPP/Jabber meetup, March 28th
+date: 2017-03-23 23:59
+author: Nÿco
+blog_id: blog
+category: Summits, Conferences, And Meetups
+---
+
+After Berlin, Stockholm, London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
+
+The Paris meetup is very simply organised by the JabberFr community, as this will be mostly oriented discussions our the modernity and contemporaneity of XMPP/Jabber.
```

Comment:
> is very simply organised

I'm not sure what "very simply" means in this context.

> modernity and contemporaneity

I'm not sure "modernity" or "contemporaneity" actually makes sense here or mean what you think they mean; also, I am a native English speaker, and I can't pronounce "contemporaneity" on the first try without getting toung tied; might want to reconsider that one :)